### ISSUE TITLE
How to use seq_tagger.py for sequence labeling

### ISSUE THREAD

**Issue by zhezhaoa** [OpenNMT/OpenNMT-tf#96]
This project is the best seq2seq  implementation of  tf version I have ever seen.
Could you provide any running examples for seq_tagger.py? e.g. it's input and output format, how to evaluate it, and so on. Since sequence labeling is very different with seq2seq tasks.

**Comment by guillaumekln** [comment]
We should indeed add some documentation on this. However, the workflow is the same as training a sequence to sequence model (see the *Quickstart* section in the main README).

You need to provide 2 parallel files with space-separated tokens, e.g for POS tagging:

```
$ head -1 ~/data/wsj/train-features.txt
Pierre Vinken , 61 years old , will join the board as a nonexecutive director Nov. 29 .
```

```
$ head -1 ~/data/wsj/train-labels.txt
NNP NNP , CD NNS JJ , MD VB DT NN IN DT JJ NN NNP CD .
```

The default `seq_tagger.py` model also requires pretrained word embeddings but you can always customize that.

By default, the evaluation will report the loss and the accuracy on the validation set. If you train on a supported tagging scheme (currently just BIOES), it can also report recall, precision, and F1-score.

Let me know if you encounter specific issues when working with sequence tagging.

**Comment by zhezhaoa** [comment]
Thank you very much! I have tried sequence_tagger and it works pretty well.

**Comment by faysalhossain2007** [comment]
can you share the latest script for this?
I am not able to run the model using the following script: `onmt-main --model_type SequenceTagger --config toy_en_de.yaml --auto_config infer tag.lua`
